src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -8096,15 +8108,18 @@ void checkWellKnown(WellKnownMember member)
             }
         }
 
-        private bool TryFindIndexOrRangeIndexerPattern(
+        /// <summary>
+        /// Finds pattern-based implicit indexer and Length/Count property.
+        /// </summary>
+        private bool TryBindIndexOrRangeImplicitIndexer(
```

Comment:
>TryBindIndexOrRangeImplicitIndexer [](http://example.com/codeflow?start=21&length=34)

The name exactly matches the name of previous function. Consider using unique name for clarity. It looks like this method binds underlying operations to perform implicit indexing. Consider reflecting this in the name. Perhaps "TryBindIndexOrRangeImplicitIndexer**Parts**" or something similar.